index.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: "Deno Documentation"
+description: "The best docs ever"
```

Comment:
I like this aspiration, but should we be more descriptive here. Especially now that it surfaces not only for search engines but also in the OG image when people share a link to our docs homepage

https://deno-docs--homepage-components.deno.dev/index.png

Reply:
Oop yeah! That's what I get for not using lorem ipsum placeholders!